bi_sql_editor/models/bi_sql_view.py

```diff
@@ -163,8 +151,6 @@ class BiSQLView(models.Model):
 
     group_ids = fields.Many2many(
```

Comment:
also, we need to inherit the mixin view and adapt those attrs' conditions

Reply:
updated